contracts/contracts/buyback/BaseBuyback.sol

```diff
@@ -152,30 +160,44 @@ abstract contract BaseBuyback is Initializable, Strategizable {
         emit TreasuryManagerUpdated(_address);
     }
 
+    /**
+     * @dev Sets the percentage of oToken to use for Flywheel tokens
+     * @param _bps BPS, 10000 to 100%
+     */
+    function setCVXShareBps(uint256 _bps) external onlyGovernor {
+        _setCVXShareBps(_bps);
+    }
+
+    function _setCVXShareBps(uint256 _bps) internal {
+        require(_bps <= 10000, "Invalid bps value");
```

Comment:
nit: custom errors are a bit nicer, especially if you need to pass a value it. Since this is not the case here, we can leave it as it is IMO